### PR TITLE
create ui list tracks by genres

### PR DESCRIPTION
**Pull request by vu9596**
Create UI list tracks by genres 
https://edu-redmine.framgia.vn/issues/2613

### CODE REVIEW

**Comment by vu9596** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Comment by NguyenMinhTuanB** [comment]
conflicted

**Comment by vu9596** [comment]
ready

File: app/src/main/res/menu/menu_genres.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item android:title="Search"
+        android:id="@+id/action_search"
+        android:icon="@drawable/ic_search"
+        app:showAsAction="ifRoom"/>
+</menu>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/menu/menu_genres.xml]: need an endline

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by vu9596** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by NguyenMinhTuanB** [comment]
merged

**Pull request merged**